include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -29,6 +29,7 @@
 #include <seqan3/argument_parser/detail/version_check.hpp>
 #include <seqan3/core/char_operations/predicate.hpp>
 #include <seqan3/core/detail/terminal.hpp>
+#include <seqan3/core/detail/test_accessor.hpp>
```

Comment:
This does not exist yet?

Reply:
I feel stupid! Forgot to add it :frowning: